### DISCUSSION TITLE
Add possibility to set MySQL connection imeout

### DISCUSSION THREAD

**Discussion by th3noname** [directus/directus#2743] [Feature Requests]
We are using Directus on ECS with Aurora serverless as a database.

We need to provide multiple test stages that don't need to run continuously. As a cost saving measure we automaticly suspend the db after a certain time.
The database needs a few seconds to resume operation, which leads to the first request failing with a connection timeout.

We would like to increase the timeout to handle these situations without errors.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
